README.md

```diff
@@ -33,7 +33,12 @@ No. It should work with any other environment as long as:
 - [pyhcl](https://pypi.org/project/pyhcl/) module installed
 - [paramiko](https://www.paramiko.org/) module installed
 - [pygit2](https://www.pygit2.org/) module installed
-- Terraform installed and configured as need to run the terraform templates you are going to use
+- Terraform
+
+paramiko is part of the base system packages. pyhcl and pygit2 are packaged in the [Uyuni utils](https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Utils/SLE_15_SP4/).
```

Comment:
```suggestion
`paramiko` is usually part of the base system packages. `pyhcl` and `pygit2` are packaged in the [Uyuni utils](https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Utils/).
```
Let's make it generic for everyone, as not everyone is using SLE15 :-)

Reply:
ok